### DISCUSSION TITLE
Getting banned using tor

### DISCUSSION THREAD

**Discussion by Benjamin5678** [rdeepak2002/reddit-place-script-2022#367] [Q&A] [answered]
Everything was working fine for me but then all of my accounts got banned at once. How is this possible if each account is using a different tor node? Is there anyway I can get around this. I was really glad that tor was added because all my accounts kept getting banned but now I guess I'm at square one.

**Comment by LDinos** [comment]
I am wondering aswell, half of my accounts got instantly banned from different ip's :(

**Comment by Benjamin5678** [comment]
What I did was stop using tor and just reduce my amount of accounts. Now that I already have what I wanted I just need enough to maintain it. Do you know what the biggest number of accounts and lowest delay is you can do without getting IP banned?

Edit: Im doing 20 accounts with delay of 15 and I think I'm good but I'm not sure.

**Comment by Benjamin5678** [comment]
I think I figured it out. Once an account changes IPs too many times it gets banned so try not to change VPN or restart tor multiple times

**Comment by Risewill23** [comment, marked as answer]
Some of the requests (That get used a lot) wasn't being sent with a proxy or tor
It's now fixed and updated

**Reply by Benjamin5678** [reply]
Has anyone successfully used tor with large amounts of bots and not gotten accounts banned?
